lib/libzfs/libzfs_mount.c

```diff
@@ -348,7 +348,7 @@ zfs_add_option(zfs_handle_t *zhp, char *options, int len,
 
 	/*
 	 * zfs_prop_get_int() to not used to ensure our mount options
```

Comment:
I know this is existing, but should this be "is not used" rather than "to not used"?